_posts/2024-10-09-call-for-editors.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: post
+date: 2024-10-09 00:00
+title:  "Call for editors"
+category:
+- announcements
+---
+
+Once again, we're looking to grow our editorial team at JOSS!
+
+Since our launch in [May 2016](https://www.arfon.org/announcing-the-journal-of-open-source-software), our [existing editorial team](http://joss.theoj.org/about#editorial_board) has handled nearly 4500 submissions (~2700 published at the time of writing, 340 under review) and the demand from the community continues to be strong. JOSS now consistently publishes 40-45 papers per month, and this is gradually increasing.
+
+New editors at JOSS are asked to make a minimum 1-year commitment, with additional years possible by mutual consent. As some of our existing editorial team are reaching the end of their term with JOSS, the time is right to bring on another cohort of editors.
+
+### Background on JOSS
+
+If you think you might be interested, take a look at our [editorial guide](https://joss.readthedocs.io/en/latest/editing.html), which describes the editorial workflow at JOSS, and also some of the reviews for [recently accepted papers](https://github.com/openjournals/joss-reviews/issues?utf8=%E2%9C%93&q=is%3Aclosed+label%3Aaccepted+). Between these two, you should be able to get a good overview of what editing for JOSS looks like.
+
+Further background about JOSS can be found in our [PeerJ CS paper](https://doi.org/10.7717/peerj-cs.147), which summarizes our first year, and our Editor-in-Chief's [original blog post](https://www.arfon.org/announcing-the-journal-of-open-source-software), which announced the journal and describes some of our core motivations for starting the journal.
+
+More recently we've also written in detail about [the costs related with running JOSS](https://blog.joss.theoj.org/2019/06/cost-models-for-running-an-online-open-journal), [scaling our editorial processes](https://blog.joss.theoj.org/2019/07/scaling), and [talked about the collaborative peer review that JOSS promotes](https://www.youtube.com/watch?v=niRiyaErqwQ).
+
+## How to apply
+
+Firstly, we especially welcome applications from prospective editors who will contribute to the diversity (e.g., ethnic, gender, disciplinary, and geographical) of our board. 
+
+✨✨✨ If you're interested in applying please fill in this [short form](https://forms.gle/SwNqHd3KYi1rwPMEA) by 9th November 2024. ✨✨✨
+
+## Who can apply
+
+Applicants should have significant experience in open source software and software engineering, as well as expertise in at least one subject/disciplinary area. Prior experience with peer-review and open science practices are also beneficial.  
+
+We are especially interested in recruiting editors in the following disciplines, although submissions from all disciplines are welcome:
+
+- Pure and/or Applied Mathematics
+- Bioinformatics
+- Computational Chemistry
+- Computer Science
```

Comment:
```suggestion
- Computer Science
- Quantum Computing
```

Reply:
I don't know what ordering we are using now... I would keep these together topically, if possible, but they don't have to be...